### ISSUE TITLE
Error when generating a PDF of an article with pictures

### ISSUE THREAD

**Issue by Atmis** [alexbevi/redmine_knowledgebase#308]
Hello,

I get the following error when I try to generate a PDF and pictures are included in the article :

Completed 500 Internal Server Error in 500ms (ActiveRecord: 15.6ms)

NoMethodError (undefined method 'number_to_human_size' for #<ArticlesController:0x0000000c964810>):
  lib/redmine/sudo_mode.rb:63:in 'sudo_mode'

Redmine is running on a Windows platform :

Environment :
  Redmine version                3.2.0.stable
  Ruby version                   2.0.0-p647 (2015-08-18) [x64-mingw32]
  Rails version                  4.2.5
  Environment                    production
  Database adapter               PostgreSQL
SCM:
  Git                            2.6.4
  Filesystem                     
Redmine plugins:
  clipboard_image_paste          1.10
  redmine_checklists             3.1.3
  redmine_dmsf                   1.5.5
  redmine_inline_note_images_plugin 0.0.1
  redmine_knowledgebase          3.1.0
  redmine_landing_page           0.1.0
  redmine_last_updated_by_column 0.3.0
  redmine_lightbox2              0.2.4
  redmine_local_avatars          0.2.0
  redmine_mail_checker           1.0.0-devel
  redmine_mail_handler_clean_body_regexp 0.1
  redmine_my_page_queries        2.1.6
  redmine_silencer               0.4.1
  redmine_spent_time             3.1.2
  redmine_stealth                0.6.0

Thank you for your help and great plugin.

**Comment by thinknot** [comment]
I just ran across the same problem, running Redmine 3.1 and the latest code from redmine_knowledgebase master branch. This might not be the best approach, but including NumberHelper worked for me. Here's the patch:
```
diff --git a/app/helpers/knowledgebase_helper.rb b/app/helpers/knowledgebase_helper.rb
index e24b1c1..3d7cf31 100644
--- a/app/helpers/knowledgebase_helper.rb
+++ b/app/helpers/knowledgebase_helper.rb
@@ -1,6 +1,7 @@
 module KnowledgebaseHelper
   include Redmine::Export::PDF
   include KnowledgebaseSettingsHelper
+  include ActionView::Helpers::NumberHelper
 
   def format_article_summary(article, format, options = {})
```

**Comment by alexbevi** [comment]
Should be fixed in v3.2.0. @Atmis @thinknot please have a look and verify when you have a moment.
